apps/app/src/jobs/tasks/integration/integration-results.ts

```
@@ -2,169 +2,173 @@ import { db } from "@bubba/db";
+
+		try {
+			// Access the integration_id to determine which integration to run
+			const integrationId = integration.integration_id;
+
+			// Extract user settings which may contain necessary credentials
+			const userSettings = integration.user_settings;
+
+			let results = null;
+
+			// Handle specific integrations using explicit case matching
+			if (1) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)